meta/devices/friendlyarm_nanopi-r1.yaml

```diff
@@ -0,0 +1,43 @@
+vendor: FriendlyARM
+model: NanoPi R1
+bootloader: unknown
+cpu:
+  model: Allwinner H3
+  cores: 4
+  mhz: 1200
+memory_mb:
+  - 512
+  - 1024
+storage:
+  - type: microSD
+device_type: WiFi Router
+usb:
+  - type: A
+    ports: 2
+ethernet:
+  - speed_mbit: 100
+    ports: 1
+    label: lan
+power_supply:
+  type: DC
+  volt: 5
+  ampere: 2
+leds:
+  - color: red
+    label: status
+  - color: green
+    label: wan
+  - color: green
+    label: lan
+keys:
+  - reset
```

Comment:
Keys should have "function" and "mode", as there are keys and switches (linux_code = EV_SW).

Reply:
```suggestion
  - function: reset
    mode: switch
```

Like this?

Reply:
Yes, that's at least the idea I have at the moment. I would however only add mode when it's not just a button, I don't know whether that makes sense in yaml context.